yarn-project/types/src/interfaces/state_provider.ts

```diff
@@ -21,14 +21,14 @@ export interface StateInfoProvider {
    * @param leafIndex - The index of the leaf for which the sibling path is required.
    * @returns The sibling path for the leaf index.
    */
-  getContractPath(leafIndex: bigint): Promise<SiblingPath<typeof CONTRACT_TREE_HEIGHT>>;
+  getContractSiblingPath(leafIndex: bigint): Promise<SiblingPath<typeof CONTRACT_TREE_HEIGHT>>;
 
   /**
-   * Returns the sibling path for the given index in the data tree.
+   * Returns the sibling path for the given index in the note hash tree.
    * @param leafIndex - The index of the leaf for which the sibling path is required.
    * @returns The sibling path for the leaf index.
    */
-  getDataTreePath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>>;
+  getNoteHashTreeSiblingPath(leafIndex: bigint): Promise<SiblingPath<typeof NOTE_HASH_TREE_HEIGHT>>;
```

Comment:
If I am pedantic, we don't need the `tree` in here, you are getting a sibling path to a note in the tree, not to a tree etc. Similar for the l1l2.

- `getNoteHashSiblingPath(leafIndex)`
- `getL1ToL2MessageSiblingPath(leafIndex)`

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3117/commits/13b3f90ebaf71a91bf3fe0d7068135515dfd2078

Reply:
No, it is not. The commit still includes the "Tree".